plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -11,16 +11,23 @@
 DOCUMENTATION = '''
 ---
 module: lxd_container
-short_description: Manage LXD Containers
+short_description: Manage LXD instances
 description:
-  - Management of LXD containers
+  - Management of LXD containers and virtual machines
 author: "Hiroaki Nakamura (@hnakamur)"
 options:
     name:
         description:
-          - Name of a container.
+          - Name of an instance.
         type: str
         required: true
+    api_endpoint:
+        description:
+          - The LXD Rest API endpoint to create containers oder virtual-machines.
+            It can be set to "/1.0/instances" for creating instances of type container or virtual-machine.
+          - If the LXD server version is older than 3.19, then this should be set to "/1.0/containers".
+        type: str
+        required: false
     architecture:
         description:
           - 'The architecture for the container (for example C(x86_64) or C(i686)).
```

Comment:
I've seen in the documentation that there are multiple lines that reference to container and not an instance. I can update this tonight, if this MR is welcome